### ISSUE TITLE
Version 0.40.0 not working with vite: SyntaxError: import not found: default

### ISSUE THREAD

**Issue by weaseldotro** [IBM/carbon-components-svelte#744]
After upgrading to version 0.40.0 I'm getting the following error: `SyntaxError: import not found: default` in `createCalendar.js:2:7`
```
import rangePlugin from "/node_modules/flatpickr/dist/plugins/rangePlugin.js?v=1300bfa4";
```
I'm using vite. This is how I can replicate the error:
- create a new project with vite https://vitejs.dev/guide/#scaffolding-your-first-vite-project
- install carbon components `npm i -D carbon-components-svelte`
- change `App.svelte` to:
```
<script lang="ts">
	import { Button } from "carbon-components-svelte"
</script>

<Button>hello</Button>
```
Then I run `npm run dev`. I open Firefox at address `http://localhost:3000/` and I get the error in the developer console.

The thing is I don't event use the `DatePicker` component in my app.

**Comment by metonym** [comment]
Add this to the list of dependencies for `vite` to optimize: `"flatpickr/dist/plugins/rangePlugin"`

**Comment by weaseldotro** [comment]
That did it, thanks.

**Comment by metonym** [comment]
Just curious, what `vite` version are you running? 

I haven't encountered this error when using it with SvelteKit, Rollup, or the official Svite set-up (`@sveltejs/vite-plugin-svelte`).

I wonder if this only affects `vite` version 1 because `@sveltejs/vite-plugin-svelte` uses `vite` version 2.

**Comment by weaseldotro** [comment]
In my project I'm using the the latest version of vite: 2.4.2. 
I replicated the error using the official guide for installing vite and svelte (see my initial message), so the latest versions I guess.

I also checked another project using the latest svelte kit and the error didn't show up.

**Comment by metonym** [comment]
Fixed in v0.40.1
